source/Ship.h

```diff
@@ -545,6 +551,7 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	Point acceleration;
 
 	int crew = 0;
+	int desiredCrew = 0;
```

Comment:
This doesn't belong on the ship model. It's a player-only consideration, and should be kept in the player model.